symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/impl/OboAuthenticatorRsaImpl.java

```diff
@@ -88,7 +90,7 @@ protected String retrieveOboSessionTokenByUsername(@Nonnull String username) thr
   protected String retrieveAppSessionToken() throws AuthUnauthorizedException {
     log.debug("Start authenticating app with id : {} ...", this.appId);
 
-    final String jwt = this.jwtHelper.createSignedJwt(this.appId, 30_000, this.appPrivateKey);
+    final String jwt = this.jwtHelper.createSignedJwt(this.appId, JwtHelper.JWT_EXPIRATION_MILLIS, this.appPrivateKey);
```

Comment:
If we always pass the same value for expiration, perhaps we can remove it from the parameter?

Reply:
I would prefer to keep this parameter. I'm not sure if we or the customers want to create a Jwt with different expiration in future, I guess it will be more flexible.

Reply:
it looks like JwtHelper it internal only so we can probably simplify it now and change it later if needed